galaxy/values.yaml

```diff
@@ -18,6 +18,1765 @@ webHandlers:
 jobHandlers:
   replicaCount: 1
 
+galaxyConf:
```

Comment:
One issue here is that we break interface compatibility with the previous version: https://github.com/CloudVE/galaxy-kubernetes/blob/80846a79a21217c859e7400b69e45191dde7fdd1/galaxy-stable/values.yaml#L43

However, I'm not sure whether we got the naming convention wrong in the previous version anyway, it looks like it shouldn't be using an underscore.